src/styles/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $padding-size;
+  gap: $gap-size;
+  width: calc(7 * #{$day-size} + 6 * #{$gap-size} + 2 * #{$padding-size});
```

Comment:
The width calculation is complex and hard to understand. Consider simplifying it or adding a comment to explain how the width is calculated. This can improve code readability and maintainability.